Example/Tests/CapturedImagesStackViewTests.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CapturedImagesStackViewTests.swift
+//  GiniVision_Tests
+//
+//  Created by Enrique del Pozo Gómez on 5/7/18.
+//  Copyright © 2018 Gini GmbH. All rights reserved.
+//
+
+import XCTest
+@testable import GiniVision
+
+final class CapturedImagesStackViewTests: XCTestCase {
+    
+    var capturedImagesStackView: CapturedImagesStackView!
+    
+    override func setUp() {
+        super.setUp()
+        capturedImagesStackView = CapturedImagesStackView(frame: .zero)
+    }
+    
+    func testCaptureStackWhenNoImages() {
+        capturedImagesStackView.updateStackStatus(to: .empty)
+        
+        XCTAssertTrue(capturedImagesStackView.isHidden,
```

Comment:
I'm not sure if a view should hide itself (from a responsibility perspective). As a standalone component it might be a strange behaviour.

On the other hand, it's quite convenient here.

Just a comment...